src/report/report.dto.ts

```diff
@@ -0,0 +1,17 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+export class SystemReportDTO {
+    @ApiProperty({ required: true })
+    title: string;
+
+    @ApiProperty({ required: true })
+    description: string;
+
+    @ApiProperty({ required: true })
+    reporterId: string;
+}
```

Comment:
No need for this field since we can get the reporter's id by JWT token (which can be access using `@UserId` in controller).